docs/install/configuration.md

```diff
@@ -282,12 +282,24 @@ If both are defined, then the **environment variables** will take priority.
     <td>Local repository for dependency loader.<br>ex)visualiztion modules of npm.</td>
   </tr>
   <tr>
+      <td><h6 class="properties">ZEPPELIN_HELIUM_NODE_INSTALLER_URL</h6></td>
+      <td><h6 class="properties">zeppelin.helium.node.installer url</h6></td>
+      <td>https://nodejs.org/dist/</td>
+      <td>Remote Node installer url for Helium dependency loader</td>
+  </tr>
+  <tr>
     <td><h6 class="properties">ZEPPELIN_HELIUM_NPM_REGISTRY</h6></td>
     <td><h6 class="properties">zeppelin.helium.npm.registry</h6></td>
     <td>http://registry.npmjs.org/</td>
     <td>Remote Npm registry for Helium dependency loader</td>
   </tr>
   <tr>
+      <td><h6 class="properties">ZEPPELIN_HELIUM_YARN_INSTALLER_URL</h6></td>
+      <td><h6 class="properties">zeppelin.helium.yarn.installer url</h6></td>
+      <td>https://github.com/yarnpkg/yarn/releases/download/</td>
+      <td>Remote Yarn installer url for Helium dependency loader</td>
```

Comment:
maybe `yarn` is a bit confusing? I guess with a lot of the projects going on here I mostly associate `yarn` with `hadoop yarn`

not sure if there is a better way to describe this though